_licenses/cern-ohl-p-2.0.txt

```diff
@@ -0,0 +1,216 @@
+---
+title: CERN Open Hardware Licence Version 2 - Permissive
+spdx-id: CERN-OHL-P-2.0
+nickname: CERN OHL v2 Permissive
+
+description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Permissive” variant allows people to take your code, relicense it and use it without any obligation to distribute the sources when they ship a product; it does have an attribution requirement.
```

Comment:
Just a suggestion to clarify the relationship of the final sentence with the text that precedes it:

```suggestion
description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Permissive” variant allows people to take your code, relicense it and use it without any obligation to distribute the sources when they ship a product; however, it does have an attribution requirement.
```

Reply:
Done, thanks